tests/abi/tools/revng-ensure-rft-equivalence/Main.cpp

```diff
@@ -0,0 +1,119 @@
+/// \file Main.cpp
+/// \brief
```

Comment:
Can you describe *in detail* what this tool does?
> This tool takes two models, looks for all the `RawFunctionType`s with the same ID...
> Using `renvg model diff` would not work since...

Reply:
Whoops. I'm sorry I forgot to fill this in.

Reply:
Done: https://github.com/revng/revng/pull/148/commits/e02a96539f4bdb67ba41bb038122743ebaa07e12.